monai/losses/__init__.py

```diff
@@ -11,6 +11,13 @@
 
 from __future__ import annotations
 
+try:
+    from ..utils import detect_default_tf32
+
+    detect_default_tf32()
```

Comment:
actually this load modules will always trigger all the submodules import https://github.com/Project-MONAI/MONAI/blob/d6bafc9e9ec09b47e87c4d57f3c31232ee13686f/monai/__init__.py#L57-L61 so probably we just need one `detect_default_tf32` in `monai/__init__.py`